### DISCUSSION TITLE
xDrip does not have full bulgarian language support

### DISCUSSION THREAD

**Discussion by DiBochev** [NightscoutFoundation/xDrip#2879] [Show and tell]
xDrip does not have full Bulgarian language support.
Most of the menus/items are in English.
Currently I am translating the missing entries in the strings-bg.xml
When I am done with this, how can I contribute to the project and give the translated xml file?

**Comment by Navid200** [comment]
Go to Crowdin and create and account and request to join the xDrip project:
https://crowdin.com/project/xdrip

**Reply by DiBochev** [reply]
I have done the registration and the join request, but still too much work (just to give you a suggestion)... 
my user is dibochev

**Comment by DiBochev** [comment]


**Reply by DiBochev** [reply]
Bulgarian language is very complicated. There are many genera of the language. To be sure everything is ok, you can send me apk file (unlocked with all menus) to check :)

If you are planning for releasing it, please send me email to: dibochev@gmail.com with version (i need it for my father)

Wish you best :)

**Reply by Navid200** [reply]
Wait for approval.
You can join as a translator.  You have to do the work.  We are all volunteers.  If you want Bulgarian language, you need to do the work.

**Reply by DiBochev** [reply]
Hi all,
The work is done :) When will be the next night/official release including latest changes related to Bulgarian language?
